content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst

```diff
@@ -16,38 +21,49 @@ process. Here's a summary of the types of lead times in Odoo:
    :align: center
    :alt: Show graphic of all lead times working together.
 
-- :ref:`Customer lead time <inventory/management/customer-lt>`: The default time frame for
+- :ref:`Customer lead time <inventory/shipping_receiving/customer-lt>`: The default time frame for
   fulfilling customer orders. The customer lead time is the number of days from the date the sales
   order (SO) is confirmed to the date the products are shipped from the warehouse. This is also
   known as *delivery lead time*.
 
-- :ref:`Sales security lead time <inventory/management/sales-security-lt>`: moves the *scheduled
-  delivery date* forward by a specified number of days. This serves as a buffer to allow the team
-  ample time to prepare the outgoing shipment earlier, considering the possibility of delays in the
-  fulfillment process.
+- :ref:`Sales security lead time <inventory/shipping_receiving/sales-security-lt>`: moves the
+  *scheduled delivery date* forward by a specified number of days. This serves as a buffer to allow
+  the team ample time to prepare the outgoing shipment earlier, considering the possibility of
+  delays in the fulfillment process.
 
-- :ref:`Purchase lead time <inventory/management/purchase-lt>`: the number of days from the
+- :ref:`Purchase lead time <inventory/shipping_receiving/purchase-lt>`: the number of days from the
   confirmation of a purchase order (PO) to the receipt of products. It provides insight on the time
   it takes for products to arrive at the warehouse, facilitating effective scheduling and planning
   of supplier deliveries.
 
-- :ref:`Purchase security lead time <inventory/management/purchase-security-lt>`: advances the order
-  deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive approach
-  of placing orders earlier mitigates the risk of vendor or shipping delays. Thus, for products that
-  are set to replenish to order, the need appears on the *Replenishment report* earlier, according
-  to the specified number of days.
+- :ref:`Purchase security lead time <inventory/shipping_receiving/purchase-security-lt>`: advances
+  the order deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive
```

Comment:
If you want you can also do a replace at the top for PO, like you did the other abbr tags